test/unit/puppet_fact_parser_test.rb

```diff
@@ -510,6 +510,11 @@ def setup
             assert_kind_of expected, subject.ram
           end
 
+          test "#kernel_version" do
+            facterversion.to_s >= '4.18.0-408.el8.x86_64' ? String : Hash
```

Comment:
what does this line do?

Reply:
right now it doesn't look like this line does anything.. Needs another look.

Reply:
@ares dug into the tests and figured out what I did wrong, updated the tests.